.script/dep.sh

```diff
@@ -8,4 +8,8 @@ go get -u github.com/colinmarc/hdfs
 go get -u golang.org/x/net/context
 go get -u google.golang.org/grpc
 go get -u github.com/golang/protobuf/proto
-go get -u github.com/MSOpenTech/azure-sdk-for-go/storage
+go get -u golang.org/x/tools/cmd/vet
```

Comment:
But I am confused that travis build failed if these two deps don't exist. 
Thus I follow the instruction of <a href = "https://github.com/Azure/azure-sdk-for-go/commit/383dafbdf92619345cb75e933524504333193f36"> this </a>